start/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeServingWindowResource.java

```diff
@@ -43,13 +49,21 @@ public Response getReady2Serve(){
     @POST
     @Path("/{orderId}")
     @Produces(MediaType.APPLICATION_JSON)
+    @Operation(operationId = "completeReadyOrder",
+               summary = "Complete a ready order",
+               description = "This operation completes an order " +
+                   "by the ServingWindow service.")
```

Comment:
Update the description to
"This operation completes an order by using the ServingWindow service."

Reply:
updated